v3/plugins/dbaeumer/vscode-eslint/2.1.3/meta.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
```

Comment:
only one version should be present (so please remove 2.1.1 and 2.1.3)

Reply:
I mentioned that in the comment
> Three versions of ES Lint were added.
> One latest stable version will be left after testing.